kernel/configs/powerpc_kata_kvm_4.19.x

```diff
@@ -191,7 +191,7 @@ CONFIG_SLUB_DEBUG=y
 # CONFIG_SLAB is not set
 CONFIG_SLUB=y
 CONFIG_SLAB_MERGE_DEFAULT=y
-# CONFIG_SLAB_FREELIST_RANDOM is not set
+CONFIG_SLAB_FREELIST_RANDOM=y
```

Comment:
Heh, so, afaict this has noting to do with random numbers, but is a security feature that randomises the freelist. Now, that is probably a good thing.... but if you do want this then I'll ask you to pull it into a separate commit ;-)